vllm/engine/async_llm_engine.py

```diff
@@ -595,6 +595,12 @@ def __init__(self,
                  log_requests: bool = True,
                  start_engine_loop: bool = True,
                  **kwargs) -> None:
+        raise ValueError(
```

Comment:
why do we unconditionally raise an error here?

Reply:
typo